docs/canonicalk8s/_parts/common_hardening.md

```diff
@@ -1,10 +1,83 @@
 ### Control plane nodes
 
+#### Control Plane Security
```

Comment:
This is not an actionable step for the how-to. I would either : 
- remove this, 
- add it to the beginning intro section and tell the user they can read it if they like 
- Read through the guide and see if there are any actionable steps from it. I gave it a preliminary run through and I think we implement most by default

Reply:
@nhennigan which beginning intro section are you thinking?  At your direction, i can just remove it

Reply:
I meant on the hardening.md file but I think it is ok to remove it. We have RBAC and x509 certs by default. This guide is for extra hardening steps we recommend you do. While adding more limitations to what different users can access is a great step in securing your cluster, I think defining more users and what permissions to assign them is out of scope of this doc.

Reply:
@nhennigan , If you're +1 with the removal.  Feel free to merge